### DISCUSSION TITLE
Use networks as embedding layer

### DISCUSSION THREAD

**Discussion by rohanishervin** [manujosephv/pytorch_tabular#379] [General]
How can I use a network from pytroch tabular and use it in another neural network. imagine I have text, image and tabular data all together. and I just want to use Tabnet for tabular part. is it possible to achieve this by using this project? I checked the documentation but I couldn't find the answer.

**Comment by manujosephv** [comment]
Two ways you can do this:
1. You can create a custom model which takes in text and image. there is a tutorial on how to implement custom models
2. Or you can use the pure pytorch models which are present for all the models implemented in the library (sadly except Tabnet, because Tabnet is just a wrapper around the tabnet library)

P.S - If you are really looking for a library for multi-modal applications, I would suggest to take a look [this](https://github.com/jrzaurin/pytorch-widedeep) library. it supports this kind of application out of the box.
